javascript/index.mjs

```diff
@@ -1,15 +1,16 @@
-module.exports = {
-  env: {
-    browser: true,
-    es2021: true
+import globals from 'globals'
+
+export const CacheESLintConfig = {
+  languageOptions: {
+    globals: {
+      ...globals.node,
+      ...globals.browser,
+      Stimulus: 'readonly'
+    }
   },
-  extends: ['eslint:recommended'],
-  parserOptions: {
-    ecmaVersion: 'latest',
-    sourceType: 'module'
-  },
-  ignorePatterns: [
-    '**/vendor/**/*.js',
+  ignores: [
+    '**/vendor/',
+    '.netlify/*'
```

Comment:
This is very project specific and shouldn't be in the global config. Also, I don't think this is something we'd want to do since it would mean we don't lint functions and edge functions.